src/components/reusable-components/searcher/form/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import lupa from '../../../../assets/lupa.png';
+
+class Form extends React.Component {
+
+  render() {
+    return (
+      <form action="" className={styles.containerBuscador}>
+        <div className={styles.containerIcon}>
+          <img src={lupa} className={styles.icon} alt="lupa" />
+        </div>
+        <input type="text" placeholder={this.props.contBuscador} className={styles.buscador} />

```

Comment:
Si `this.props.contBuscador` es un placeholder, sería apropiado que esa prop se llame igual ya que haría más fácil entender para qué sirve.
```suggestion
        <input type="text" placeholder={this.props.placeholder} className={styles.buscador} />
```